python/lsst/ts/cbp/component.py

```diff
@@ -337,7 +337,7 @@ async def check_park(self):
         """
         self.park = bool(await self.send_command("park=?"))
 
-    async def set_park(self, park: int = 0):
+    async def set_park(self):
```

Comment:
Please update the doc string, since this no longer unparks.

Also please use active voice (per PEP-8) and consider removing "A function that". 

I suggest something simple, such as `"""Park the CBP."""`.